src/components/Input/Input.tsx

```diff
@@ -4,14 +4,17 @@ import { ClassValue } from 'classnames/types';
 
 import styles from './Input.module.css';
 
-interface InputProps {
+interface InputProps
+  extends Omit<React.InputHTMLAttributes<HTMLInputElement>, 'className'> {
   className?: ClassValue;
```

Comment:
Unrelated: Not actually sure we should do this the more I think about it. Probably should stick with the default since  we don't necessarily need to expect `className` to be anything other than `string`. I think that should be up to the consumer how to generate the class string here. Thoughts? (@pixelbandito @dkordik)

Reply:
Can you elaborate on what the "right" way to do it is?

Are you just saying stick with `className?: string;` I'd be down with that.

I have no interest in supporting consumers passing ClassMaps into these components, they can make their own darn string.

Reply:
If we want to stick with the `className?: string`, then we don't have to do anything at all in this case since that is what is provided as part of `React.InputHTMLElement` and other similar interfaces provided by React.